std_msgs/Header header
float32 FXpMZp
float32 FXpMZm
float32 FXmMZp
float32 FXmMZm
float32 FYpMZp
float32 FYpMZm
float32 FYmMZp
float32 FYmMZm
